client/state/automated-transfer/constants.js

```diff
@@ -1,7 +1,17 @@
-export const AUTOMATED_TRANSFER_STATUS = {
+export const transferStates = {
+	INQUIRING: 'inquiring',
```

Comment:
@copons what do you think about this? should we continue to leave the status as `null` when we're checking for eligibility?